packages/javascript-sdk/tsconfig.lib.json

```diff
@@ -12,7 +12,7 @@
     "module": "ES2020",
     "target": "ES2020"
   },
-  "include": ["**/*.ts"],
+  "include": ["**/*.ts", "jest.setup.js"],
```

Comment:
I think I addedt his when I was trying to find ways to test this with jest, but I can remove it.

Reply:
removed